server/hangouts/methods.js

```diff
@@ -404,3 +404,36 @@ Meteor.methods({
 
   }
 });
+
+Meteor.methods({
```

Comment:
Please Move methods related to `AppStats` collection under a  `/server/app_stats/methods.js` directory.
`joinParticipant` and `leaveParticipant`

Reply:
Fixed